definitions/npm/algoliasearch_v3.x.x/test_algoliasearch-v3.x.x.js

```diff
@@ -1,8 +1,25 @@
 // @flow
-import algoliasearch from 'algoliasearch';
+import algoliasearchImport from 'algoliasearch';
```

Comment:
Can you use the new `describe` and `it` blocks for extra winning? More in the CONTRIBUTING.md file. :)

Reply:
Will do.